src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml

```diff
@@ -222,28 +326,64 @@
             </vsimaging:CrispImage>
         </Button>
 
-        <!-- Status Text and Refresh Button -->
+        <!-- Advanced Settings -->
+        <Expander x:Name="advancedSettingsExpander"
+                  x:Uid="uidadvancedSettingsExpander"
+                  HorizontalAlignment="Stretch"
+                  Header="{x:Static local:UIResources.AdvancedSettingsHeader}"
+                  Grid.Row="2"
+                  Grid.ColumnSpan="2"
+                  Margin="0,10,0,0"
+                  VerticalAlignment="Top"
+                  ExpandDirection="Down">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto" />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="Auto" />
+                    <ColumnDefinition Width="*" />
+                </Grid.ColumnDefinitions>
+                
+                <Label x:Uid="uidHostnameLabel"
+                       Grid.Row="0"
+                       Grid.Column="0"
+                       x:Name="Hostname"
+                       Target="{Binding ElementName=hostnameTextBlock}"
+                       Margin="19,5,0,5"
+                       Content="{x:Static local:UIResources.HostnameLabel}"
+                       ToolTip="{x:Static local:UIResources.HostnameTip}" />
 
-        <TextBlock x:Uid="uidStatusLabel"
-                    Style="{StaticResource StatusLabelStyle}"
-                    Grid.Row="2"
-                    Grid.Column="0"
-                    Margin="0,10,0,10"
-                    Padding="10,0,10,0"
-                    Text="{ Binding StatusText, Mode=OneWay}"
-                    TextWrapping="Wrap"/>
+                <TextBox x:Uid="uidHostnameTextBlock"
+                         Grid.Row="0"
+                         Grid.Column="1"
+                         x:Name="hostnameTextBlock"
+                         Text="{Binding Path=Hostname, Mode=TwoWay}"
```

Comment:
Should this binding have and UpdateSourceTrigger of TextChanged instead of the default of LostFocus?